IOSExercises/Classes/LocaleManager.h

```diff
@@ -0,0 +1,31 @@
+//
+//  LocaleManager.h
+//  IOSExercises
+//
+//  Created by Arnold Joseph Caesar Esteban on 11/4/20.
+//  Copyright © 2020 Arnold Joseph Caesar Esteban. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+typedef NSString * SupportedRegion NS_STRING_ENUM;
```

Comment:
For now, please use integer values in enum types (for Swift detection):

typedef NS_ENUM(NSUInteger, EXRSupportedRegion) {
  EXRSupportedRegionDefault = 0,
  EXRSupportedRegionUS,
  ...
};

Strings will be declared as constants:
static NSString * const kEXRSupportedRegionUS = @"US";

Also use prefix since name is general and might conflict with Apple's internal types

And declare in `.m` file unless constants/enums are needed by other classes

Reply:
Re prefix: Please use only if you also prefix the object/class name. Otherwise no concerns to keep as is